SpatialGDK/Source/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -163,7 +163,7 @@ void USpatialNetDriver::OnConnected()
 	PackageMap = Cast<USpatialPackageMapClient>(GetSpatialOSNetConnection()->PackageMap);
 
 	View->Init(this);
-	Sender->Init(this);
+	Sender->Init(this, TimerManager);
```

Comment:
Nit: NetDriver owns the TimerManager. Maybe just get the TimerManager from there rather than passing?

Reply:
`NetDriver` doesn't own it though, `GameInstance` does.
Actually, we currently expose all the things `SpatialNetDriver` owns as `public`. Maybe we should instead make them private and friend the classes like `Sender` and `Receiver` (and whatever else actually needs access to those)?